### ISSUE TITLE
Getting the Token from body and not from header

### ISSUE THREAD

**Issue by blackw212** [d0whc3r/vue-auth-plugin#118]
Hello,

I would like to use your plugin in my next personal project, as it simplifies things by a lot.
Yet i am curious, how could i achieve that once the Response message from the API server is sent, that i could grab the Token from the body? Currently having some issues in providing custom headers as response, which include the access token generated.

If i wasn't clear enough, please let me know and i will further explain!

Thank you in advance !

**Comment by d0whc3r** [comment]
Hi,
Now is no possible to get auth token from body. Token is get from header response.
Explain me what do you need and I take a look if I can adjust the code to do that

**Comment by blackw212** [comment]
Hello,

Thank you for your reply! What i mean is that, after i make the POST request towards the server, i would like to get the Token from the Data received. I will add a screenshot to make it easier to understand.
![res](https://user-images.githubusercontent.com/5298004/59967026-e1bafc80-9524-11e9-8a1c-b5a902f9a519.PNG)

As seen there, i have the access token returned in the Data. I would like to be able to grab it from there.

**Comment by blackw212** [comment]
So hello once again,

Just realised that the header gets stripped down to that by VueJS when i try to console log the header from your Plugin. Is this normal? 

If i use Postman to check my request, the Authorisation header is there and exists.

**Comment by blackw212** [comment]
To get things more clear, this is the response i get using an Restapi Client (similar to Postman or so):
![header_postman](https://user-images.githubusercontent.com/5298004/60022656-89a30800-9694-11e9-8d52-4002cfd0f038.PNG)


And the image in my previous post, is what i get when i do the following: 
`                    this.$auth.login({
                        username: this.username,
                        password: this.password,
                    }).then((response) => {
                        // eslint-disable-next-line
                        console.log(response);
                    })`
Basically console logging the response i get after the promise.

Am i missing something here? This is driving me nuts...

**Comment by d0whc3r** [comment]
Sorry for waiting
https://github.com/d0whc3r/vue-auth-plugin/pull/128
now you could get the token from where you want
using `customToken` function defined in `loginData` option.
customToken is a function that receives the response from axios login petition and must return a string that is the token to use

**Comment by d0whc3r** [comment]
changes are available in version 1.3.0
